pkg/client/workflows.go

```diff
@@ -68,14 +68,18 @@ func newWorkflows(c *ThirdPartyClient, namespace string) *workflows {
 	}
 }
 
+// Update updates the given workflow in the cluster
 func (w *workflows) Update(workflow *api.Workflow) (result *api.Workflow, err error) {
 	return w.UpdateWithSubresource(workflow, "")
 }
 
+// UpdateStatus updates the status field for a given workflow TODO: Is this correct?
```

Comment:
But it is identical to the function 'Update' right above it? shouldnt the UpdateWithSubresource call have something other than the empty string as second argument?